xc/common/utils/prjxray_routing_import.py

```diff
@@ -1019,6 +1023,20 @@ def get_number_graph_edges(conn, graph, node_mapping):
         if dest_graph_node not in node_mapping:
             continue
 
+        src_node, src_node_type = node_mapping[src_graph_node]
```

Comment:
Need a comment on what this is going?

Reply:
Consider factoring in a function?

Reply:
Factored in a function and added a docstring to it